Classes/Labels/LabelsViewController.swift

```diff
@@ -19,6 +19,8 @@ LabelSectionControllerDelegate {
     private let client: GithubClient
     private let request: RepositoryLabelsQuery
 
+    var wasDismissedByDone: Bool = false
```

Comment:
The `: Bool` is inferred; you can omit it!

Reply:
Done